### PR TITLE
Adding Odysee Channels to Media

### PR DESCRIPTION
**Pull request by w1g0**

### CODE REVIEW

**Comment by w1g0** [comment]
Does it fail because this icon is missing? 🤔
Odysee

**Comment by IstoraMandiri** [comment]
The icons are from a list of SVG items, which probably doesn't have the odysee logo.

For now, we can just use `video` icon

**Comment by IstoraMandiri** [comment]
By the way, do you know who owns these channels and their purpose?

Once the community calls channel reaches 300 subs it's possible to auto-sync with YouTube, how do we plan around that?

Also can we remove the 'invite' link and just link to the actual channel?

**Comment by w1g0** [comment]
> By the way, do you know who owns these channels and their purpose?
> 
> Once the community calls channel reaches 300 subs it's possible to auto-sync with YouTube, how do we plan around that?
> 
> Also can we remove the 'invite' link and just link to the actual channel?

Yes, I'm the owner of the channel. The purpose of Ethereum_Classic is in accordance with the Eth_Classic twitter handle and EthereumClassicNews, like ETC_Network is used to repost community contributions.

The reason that the invite link is used is because it supports the channel with $LBC credits, these credits are used to upload and edit videos. If requested that can be changed but then it becomes spending my credits (which has always been like this), while it is already my energy spend here to maintain. This is not like YouTube where you can monetize the system and get rewarded by the algorithm. If I want to have more visibility it requires more credits. 
Basically leave the invite link for more chance on success or replace it and push it off chain but I sense that YouTube is being pushed due to its network effect, keep boiling the frog. Until now I haven't seen an integrated donation culture in ETC.

**Comment by IstoraMandiri** [comment]
As the https://odysee.com/@Ethereum_Classic:2 channel is just community calls, shouldn't it be named as such in the link?

Also, in lieu of a logo, could we addd `(Odysee)` to the end of the link name for clarity?

**Comment by w1g0** [comment]
> As the https://odysee.com/@Ethereum_Classic:2 channel is just community calls, shouldn't it be named as such in the link (instead of "Ethereum Classic Announcements & Updates")?
> 
> I will check if the library of logos has an Odysee icon and get update this issue.
> 
> We should also probably add "(Playlist)" to the ETC for Investors playlist.
> 
> As there are more channels being added we should try to make them described uniquely to help people navigate.

It started out by co-hosting the Core Dev Calls, so I would like to keep it an open format for any videos that are being submitted by a subset of verified members on rough consent. Not limit it into Community Calls and keep an open approach.

Yes, moving it to another chapter would make sense. Maybe also rename Twitter to Microblogging so we can add Mastodon accounts under the same umbrella?

**Review by gitr0n1n — CHANGES_REQUESTED** [review]
Resubmit PR due to internationalization PR changing the edited file and creating a conflict per @IstoraMandiri 's previous notes

**Comment by IstoraMandiri** [comment]
I found it was also possible to pull the updates into this branch without creating a new PR.

Anyways, I'll try and implement these changes in an update soon.

**Comment by IstoraMandiri** [comment]
Unfortunately there's no Odysee logo in the SVG pack we use yet, so I've left it at video, I added ~(Odysee)` to the end of the name for now.

I merged master into this branch, so it can now be merged back in.

I renamed "Media" to "Video Channels & Playlists", so this section is clearer.

**Review by IstoraMandiri — APPROVED** [review]

**Comment by w1g0** [comment]
> Unfortunately there's no Odysee logo in the SVG pack we use yet, so I've left it at video, I added ~(Odysee)` to the end of the name for now.
> 
> I merged master into this branch, so it can now be merged back in.
> 
> I renamed "Media" to "Video Channels & Playlists", so this section is clearer.
> 
> I renamed "Ethereum Classic" to "ETC" in all of those playlists so that it's easier to read.

The video icon would make more sense this point in time since Odysee services are lesser known and many won't "recognize" the icon anyways. When the SVG pack is updated we can update it.

**Review by gitr0n1n — APPROVED** [review]
